### ISSUE TITLE
FilWip crashes on "Preferences" (64 bit)

### ISSUE THREAD

**Issue by Vidrep** [HaikuArchives/FilWip#20]
The application will crash when clicking on FilWip preferences.
Debug report pasted below:

```
Debug information for team /boot/system/apps/FilWip/FilWip (883):
CPU(s): 4x Intel Core™ i5-3570
Memory: 7.95 GiB total, 360.39 MiB used
Haiku revision: hrev51310 Jul 27 2017 09:45:50 (x86_64)

Active Threads:
	thread 887: Bubble_Helper 
	thread 888: The Eraser 
	thread 889: _Previous system state 
	thread 890: _Temporary files 
	thread 891: _Trashed items 
	thread 892: _Custom queries 
	thread 893: _System log 
	thread 894: _Bash History 
	thread 895: _HaikuDepot cache 
	thread 896: _locatedb 
	thread 897: _Cache 
	thread 898: _Cookies 
	thread 899: _History 
	thread 900: _Site icons 
	thread 901: _Session 
	thread 902: w>FilWip 
	thread 904: Bubble 
	thread 906: team 883 debug task 
	thread 883: FilWip (main)
		state: Exception (Segment violation)

		Frame		IP			Function Name
		-----------------------------------------------
		0x7f9bf35360f0	0x65a1dfc093	strdup + 0x21 
			Disassembly:
				strdup:
				0x00000065a1dfc072:           4885ff  test %rdi, %rdi
				0x00000065a1dfc075:             7447  jz 0xa1dfc0be
				0x00000065a1dfc077:               55  push %rbp
				0x00000065a1dfc078:           4889e5  mov %rsp, %rbp
				0x00000065a1dfc07b:             4155  push %r13
				0x00000065a1dfc07d:             4154  push %r12
				0x00000065a1dfc07f:               53  push %rbx
				0x00000065a1dfc080:         4883ec08  sub $0x8, %rsp
				0x00000065a1dfc084:           4989fd  mov %rdi, %r13
				0x00000065a1dfc087:       b800000000  mov $0x0, %eax
				0x00000065a1dfc08c:   48c7c1ffffffff  mov $0xffffffffffffffff, %rcx
				0x00000065a1dfc093:             f2ae  repne scasb  <--

			Frame memory:
				[0x7f9bf35360c0]  .`S....... J....   e0 60 53 f3 9b 7f 00 00 90 9c 20 4a cd 00 00 00
				[0x7f9bf35360d0]  ..(J.....4.uN...   f0 8a 28 4a cd 00 00 00 d4 34 f8 75 4e 01 00 00
				[0x7f9bf35360e0]  .aS.......p.N...   00 61 53 f3 9b 7f 00 00 9d ff 70 bb 4e 00 00 00
		0x7f9bf3536110	0x4ebb70ff98	PrefsListItem::PrefsListItem(char const*) + 0x28 
		0x7f9bf3536300	0x4ebb711bab	PrefsWindow::PrefsWindow() + 0x2bb 
		0x7f9bf3536320	0x4ebb7039f0	FilWip::MessageReceived(BMessage*) + 0xd0 
		0x7f9bf3536330	0x14e75f2e1fd	BLooper::DispatchMessage(BMessage*, BHandler*) + 0x31 
		0x7f9bf3536570	0x14e75f276d4	BApplication::DispatchMessage(BMessage*, BHandler*) + 0x396 
		0x7f9bf35365d0	0x14e75f2e586	BLooper::task_looper() + 0x1d6 
		0x7f9bf35365f0	0x14e75f243a8	BApplication::Run() + 0x4a 
		0x7f9bf3536610	0x4ebb70011a	main + 0x2a 
		0x7f9bf3536640	0x4ebb700349	_start + 0x4f 
		0x7f9bf3536660	0x194347d7e46	runtime_loader + 0x129 
		00000000	0x7f8154e58260	commpage_thread_exit + 0 

```

**Comment by Vidrep** [comment]
I built FilWip from latest sources for both x86_gcc2h and x86_64 (FilWip-1.0.0-8).
Only 64 bit is crashing when opening the preferences panel. The one time it did not crash, the preferences selection window was empty. I will post screenshots of both.
Below is the relevent part of the syslog when it crashes.

KERN: vm_page_fault: vm_soft_fault returned error 'Permission denied' on fault at 0xffffffffffffffff, ip 0xeb30fce093, write 0, user 1, thread 0x33a
KERN: vm_page_fault: thread "FilWip" (826) in team "FilWip" (826) tried to read address 0xffffffffffffffff, ip 0xeb30fce093 ("???" +0xeb30fce093)
KERN: debug_server: Thread 826 entered the debugger: Segment violation
KERN: stack trace, current PC 0xeb30fce093  strdup + 0x21:
KERN:   (0x7ff61d1a81e0)  0x161b83baf9d  _ZN13PrefsListItemC2EPKc + 0x2d
KERN:   (0x7ff61d1a8200)  0x161b83bcbb0  _ZN11PrefsWindowC2Ev + 0x2c0
KERN:   (0x7ff61d1a83f0)  0x161b83ae9f5  _ZN6FilWip15MessageReceivedEP8BMessage + 0xd5
KERN:   (0x7ff61d1a8410)  0x17c37dfe0fa  _ZN12BApplication15DispatchMessageEP8BMessageP8BHandler + 0x2a
KERN:   (0x7ff61d1a8660)  0x17c37e06013  _ZN7BLooper11task_looperEv + 0x123
KERN:   (0x7ff61d1a86c0)  0x17c37dfb207  _ZN12BApplication3RunEv + 0x47
KERN:   (0x7ff61d1a86e0)  0x161b83ab11f  main + 0x2f
KERN:   (0x7ff61d1a8700)  0x161b83ab34e  _start + 0x54
KERN:   (0x7ff61d1a8730)  0x18bf1c01e48  runtime_loader + 0x12b
KERN: debug_server: Killing team 826 (/boot/system/apps/FilWip/FilWip)
KERN: debug_server: TeamDebugHandler::Init(): Failed to get info for team 826: Operation on invalid team
KERN: debug_server: KillTeam(): Error getting info for team 826: Operation on invalid team
KERN: debug_server: Killing team 826 ()

**Comment by Vidrep** [comment]
![filwip-1 0 0-8-x86_64](https://user-images.githubusercontent.com/13264393/28803597-ed9b6c36-761a-11e7-9e8c-b39fbf0d94b1.png)
![filwip-1 0 0-8-x86_gcc2h](https://user-images.githubusercontent.com/13264393/28803599-f4b99ae2-761a-11e7-8400-820e2f28e31f.png)

**Comment by Vidrep** [comment]
This issue has been resolved with update to v0.1.1. Thanks!
